package.json

```diff
@@ -34,7 +34,7 @@
     "style-loader": "0.18.2"
   },
   "scripts": {
-    "start": "node scripts/start.js",
+    "start": "styleguidist server",
```

Comment:
you can remove this line since we already have `docs` command. If you want to keep the `start`, then remove `docs`.

Reply:
The issue is when I try `npm docs`, it goes to github toolset page. I have to use `npm run docs`. As I am a little bit lazy, so I put it into start. ^_^ If you think it is OK, then I can remove `docs`.

Reply:
okay. First of all, you can not run with `npm docs` or `yarn docs`. Only `start` and `test` can run by `npm start`, `yarn start` and `npm test`, `yarn test`. Others should be run by `npm run docs` or `yarn run docs`. You need to have `run` after the `npm` or `yarn`